src/name-resolver.ts

```diff
@@ -72,6 +72,9 @@ export class NameResolver extends Logger {
    */
   public async getAddressOrContent(name: string, type: string): Promise<string> {
     this.log(`looking up ENS name "${name}"`, 'debug');
+    if (!name) {
+      throw new Error('ens name is undefined. Provide valid ens name. example ens name: xyz.evan');
```

Comment:
```suggestion
      throw new Error('ENS name is undefined. Provide valid ENS name. Example ENS name: xyz.evan');
```

Reply:
Also, if you choose to apply this suggestion, keep in mind to adapt the expected error messages in the tests